std/algorithm/mutation.d

```diff
@@ -2389,6 +2378,120 @@ void swap(T)(ref T lhs, ref T rhs) if (is(typeof(lhs.proxySwap(rhs))))
     lhs.proxySwap(rhs);
 }
 
+/**
+Swaps two elements in-place of a random access range `r`,
```

Comment:
@greenify By "fix the docs", we meant to change `a random access range` here to just `a range`, because `random access` wrongly implies that `isRandomAccessRange!R` is required.